### PR TITLE
Client Server support

### PR DESCRIPTION
**Pull request by xzdandy**
For easy performance testing and usage. 

Done: 
- Eva Server
- Command line Eva Client
- Server Response data model
- Streaming-based transmission for large-size response

TODO:  metrics 

Example Usage:
python eva.py

python eva_client.py # Below two queries will take a while.
LOAD DATA INFILE 'data/ua_detrac/ua_detrac.mp4' INTO MyVideo;
SELECT id, data FROM MyVideo WHERE id < 5;

### CODE REVIEW

File: src/server/interpreter.py [hunk 1]
```diff
@@ -13,84 +13,62 @@
...
-                    frame_name = output_folder + "output" + str(i) + ".jpg"
-                    ouput_frames[i].save(frame_name)
-
-                print("Refer pop-up for a sample of the output")
-                ouput_frames[0].show()
-
-            except TypeError:
-                print("SQL Statement improperly formatted. Try again.")
+        self._protocol._response_chunks = []
+        self._protocol.send_message(query)
```
Comment (gaurav274) [inline src/server/interpreter.py:51]: Remove the prints

File: test/models/server/test_response.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import unittest
+
+from src.models.storage.batch import Batch
+from src.models.server.response import ResponseStatus, Response
+from test.util import create_dataframe
+
+
+class ResponseTest(unittest.TestCase):
+
+    def test_server_response_to_json_string(self):
```
Comment (gaurav274) [inline test/models/server/test_response.py:24]: I think this test can be removed

File: test/models/storage/test_batch.py [hunk 3]
```diff
@@ -23,6 +23,17 @@
 
 class BatchTest(unittest.TestCase):
 
+    def test_batch_to_json(self):
```
Comment (gaurav274) [inline test/models/storage/test_batch.py:26]: This can be removed

**Review by gaurav274 — APPROVED** [review]
LGTM

**Comment by xzdandy** [comment]
Query can be very long. Multiple line input support in client command line can be helpful.

**Comment by xzdandy** [comment]
Create a new pull request for this.

**Pull request closed without merging**